drivers/wifi/eswifi/eswifi_bus_spi.c

```diff
@@ -245,7 +245,7 @@ int eswifi_spi_init(struct eswifi_dev *eswifi)
 		return -ENODEV;
 	}
 	spi->dr.pin = DT_INVENTEK_ESWIFI_ESWIFI0_DATA_GPIOS_PIN;
-	gpio_pin_configure(spi->dr.dev, spi->dr.pin, GPIO_DIR_IN);
+	gpio_pin_configure(spi->dr.dev, spi->dr.pin, GPIO_INPUT);
```

Comment:
`GPIO_INPUT | DT_INVENTEK_ESWIFI_ESWIFI0_DATA_GPIOS_FLAGS`

Reply:
It looks like the `DT_INVENTEK_ESWIFI_ESWIFI0_DATA_GPIOS_FLAGS` flag is still missing.

Reply:
my bad.. fixed, thanks. Seems like some coffee will be welcome .